src/daft-connect/src/translation/expr/unresolved_function.rs

```diff
@@ -24,12 +24,36 @@ pub fn unresolved_to_daft_expr(f: &UnresolvedFunction) -> eyre::Result<daft_dsl:
 
     match function_name.as_str() {
         "count" => handle_count(arguments).wrap_err("Failed to handle count function"),
+        "<" => handle_binary_op(arguments, daft_dsl::Operator::Lt)
+            .wrap_err("Failed to handle < function"),
+        ">" => handle_binary_op(arguments, daft_dsl::Operator::Gt)
+            .wrap_err("Failed to handle > function"),
+        "<=" => handle_binary_op(arguments, daft_dsl::Operator::LtEq)
+            .wrap_err("Failed to handle <= function"),
+        ">=" => handle_binary_op(arguments, daft_dsl::Operator::GtEq)
+            .wrap_err("Failed to handle >= function"),
```

Comment:
Any reason we're using `eyre` here instead of the same error handling mechanisms as the rest of the codebase? Looking through the codebase it seems like it's a unique dependency to daft-connect.

Reply:
@jaychia I talked to @universalmind303 about this. generally `anyhow/eyre` is preferred for binaries / internal usage and `thiserror/snafu` for libs. I would love to talk about it more but think it could be done better outside of an issue.

Reply:
Gotcha, thanks yeah it isn't blocking (PR has been green-lighted and approved already), was just a question